src/plugins/intel_cpu/src/mkldnn_exec_network.cpp

```diff
@@ -423,11 +423,9 @@ bool MKLDNNExecNetwork::canBeExecViaLegacyDynBatch(std::shared_ptr<const ov::Mod
 
         if (type == Tile) {
             const auto tile = std::dynamic_pointer_cast<const ngraph::opset1::Tile>(op);
```

Comment:
Could you please move `tile` initialization closer to its usage in the `if` statement?